packages/mobile/src/components/seller-info/seller-info.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { FC } from 'react';
+import { globalStyles } from '~/styles/styles';
+import { ButtonAppearance } from '~/common/enums/enums';
+import { ColorPalette } from '@vse-bude/shared';
+import { MOCK_IMAGES } from '~/mock/mock-images';
+import { useTranslation } from 'react-i18next';
+import {
+  Text,
+  View,
+  Image,
+  Pressable,
+  SecondaryButton,
+  FacebookIcon,
+  PhoneIcon,
+  InstagramIcon,
+  GlobeIcon,
+} from '../components';
+import { styles } from './styles';
+
+const SellerInfo: FC = () => {
```

Comment:
It is supposed to receive a `seller` prop  and then take all of the data from there.

Reply:
fixed for now with mock data